cli/src/init.rs

```diff
@@ -189,52 +187,7 @@ fn replace_features(file_path: &str, adapters: Vec<&str>) -> anyhow::Result<()>
     replace_string_in_file(file_path, target, &features.join(", "))
 }
 
-fn circom_lib_template(file_path: &str) -> anyhow::Result<()> {
```

Comment:
The following functions were moved to `circom.rs` or `halo2.rs` under `init` folder respectly.